src/drt/test/gcTest.cpp

```diff
@@ -388,6 +388,17 @@ BOOST_DATA_TEST_CASE(spacing_prl,
 BOOST_DATA_TEST_CASE(design_rule_width, bdata::make({true, false}), legal)
 {
   // Setup
+  auto dbLayer = odb::dbTechLayer::create(tech, "m1", odb::dbTechLayerType::ROUTING);
+  dbLayer->initTwoWidths(2);
+  dbLayer->addTwoWidthsIndexEntry(90);
+  dbLayer->addTwoWidthsIndexEntry(190);
+  dbLayer->addTwoWidthsSpacingTableEntry(0,0,0);
+  dbLayer->addTwoWidthsSpacingTableEntry(0,1,50);
+  dbLayer->addTwoWidthsSpacingTableEntry(1,0,50);
+  dbLayer->addTwoWidthsSpacingTableEntry(1,1,150);
+  frTechObject* tech = design->getTech();
+  frLayer* layer = tech->getLayer(2);
+  layer->setDbLayer(dbLayer);
   makeSpacingTableTwConstraint(2, {90, 190}, {-1, -1}, {{0, 50}, {50, 100}});
```

Comment:
Does it make sense to move the dbLayer setup into makeSpacingTableTwConstraint ?

Reply:
That would require me to move dbTech to fixture. It can be done and should be since eventually all constraints should point to their odb equivalent. However, I prefer to do it in a separate PR because I will be changing dependency on dbTech in test.